platforms/ide/tooling-api/src/crossVersionTest/groovy/org/gradle/integtests/tooling/r940/FetchBuildActionCrossVersionSpec.groovy

```diff
@@ -0,0 +1,255 @@
+/*
+ * Copyright 2025 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.integtests.tooling.r940
+
+import org.gradle.integtests.tooling.fixture.TargetGradleVersion
+import org.gradle.integtests.tooling.fixture.ToolingApiSpecification
+import org.gradle.integtests.tooling.fixture.ToolingApiVersion
+import org.gradle.integtests.tooling.r16.CustomModel
+import org.gradle.util.GradleVersion
+import org.gradle.integtests.tooling.r930.FetchBuildActionCrossVersionSpec.FetchCustomModelPerProjectAction
+import org.gradle.integtests.tooling.r930.FetchBuildActionCrossVersionSpec.FetchCustomModelAction
+import org.gradle.integtests.tooling.r930.FetchBuildActionCrossVersionSpec.FetchGradleBuildAction
+import org.gradle.integtests.tooling.r930.FetchBuildActionCrossVersionSpec.FetchUnknownModelAction
+
+@ToolingApiVersion(">=9.3.0")
+@TargetGradleVersion(">=9.4.0")
+class FetchBuildActionCrossVersionSpec extends ToolingApiSpecification {
```

Comment:
❓ Is that a copy of `org.gradle.integtests.tooling.r930.FetchBuildActionCrossVersionSpec`?